ingestion/functions/parsing/brazil_espirito_santo/espirito_santo.py

```diff
@@ -0,0 +1,326 @@
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            "common/python"))
+    import parsing_lib
+
+_AGE = "FaixaEtaria"
+_GENDER = "Sexo"
+_MUNICIPALITY = "Municipio"
+_DATE_CONFIRMED = "DataNotificacao"
+_DATE_DEATH = "DataObito"
+_CLASSIFICATION = "Classificacao"
+_OUTCOME = "Evolucao"
+_LUNG = "ComorbidadePulmao"
+_KIDNEY = "ComorbidadeRenal"
+_DIABETES = "ComorbidadeDiabetes"
+_OBESITY = "ComorbidadeObesidade"
+_CARDIOVASCULAR = "ComorbidadeCardio"
+_SMOKER = "ComorbidadeTabagismo"
+_PCR = "DataColeta_RT_PCR"
+_SEROLOGICAL_METHOD1 = "DataColetaTesteRapido"
+_SEROLOGICAL_METHOD2 = "DataColetaSorologia"
+_SEROLOGICAL_METHOD3 = "DataColetaSorologiaIGG"
+_TEST_CLASS = "CriterioConfirmacao"
+_ETHNICITY = "RacaCor"
+_NEIGHBOURHOOD = "Bairro"
+_FEVER = "Febre"
+_DIFFICULTY_BREATHING = "DificuldadeRespiratoria"
+_COUGH = "Tosse"
+_SORE_THROAT = "DorGarganta"
+_DIARRHOEA = "Diarreia"
+_HEADACHE = "Cefaleia"
+_CORYZA = "Coriza"
+_HOSPITALIZED = "FicouInternado"
+_INTERNAL_TRAVEL = "ViagemBrasil"
+_INTERNATIONAL_TRAVEL = "ViagemInternacional"
+_HEALTHCARE_PROFFESIONAL = "ProfissionalSaude"
+_HOMELESS = "MoradorDeRua"
+_DISABILITY = "PossuiDeficiencia"
+_SCHOOLING = "Escolaridade"
+
+
+_COMORBIDITIES_MAP = {
+    "ComorbidadePulmao": "lung disease",
+    "ComorbidadeRenal": "kidney disease",
+    "ComorbidadeDiabetes": "diabetes mellitus",
+    "ComorbidadeObesidade": "obesity",
+    "ComorbidadeCardio": "cardiovascular system disease"
+}
+
+
+_EDUCATION_MAP = {
+    "1ª a 4ª série incompleta do EF (antigo primário ou 1º grau)": "First four years of elementary school not completed",
+    "Ensino fundamental completo (antigo ginásio ou 1º grau) ": "Elementary school completed",
+    "Ensino médio completo (antigo colegial ou 2º grau ) ": "High school completed",
+    "5ª à 8ª série incompleta do EF (antigo ginásio ou 1º grau)": "Years 5 to 8 of elementary school not completed",
+    "Ensino médio incompleto (antigo colegial ou 2º grau )": "High school not completed",
+    "Educação superior incompleta ": "Higher education not completed",
+    "Educação superior completa": "Higher education completed",
+    "Analfabeto": "Analphabet",
+    "4ª série completa do EF (antigo primário ou 1º grau)": "Fourth year of elementary school completed"
+}
+
+
+_NONE_TYPES = set(["Não", "Ignorado", "-", "Não Informado", "Não se aplica", None])
+
+
+_SYMPTOMS_MAP = {
+    "Cefaleia": "headache",
+    "DorGarganta": "throat pain",
+    "Febre": "fever",
+    "Tosse": "cough",
+    "Diarreia": "diarrhoea",
+    "DificuldadeRespiratoria": "dyspnea"
+}
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+    """
+    if raw_date:
+        date = datetime.strptime(raw_date, "%Y-%m-%d")
+        return date.strftime("%m/%d/%YZ")
+
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "M":
+        return "Male"
+    if raw_gender == "F":
+        return "Female"
+
```

Comment:
I think we've really reached the point where functions like this should be in a common library, as multiple parsers all do this.

Reply:
Yes I agree, it's next on my list of things I promise